backend/shelf-server/src/main/java/com/htec/shelfserver/service/TokenService.java

```diff
@@ -87,6 +87,10 @@ public String createAndSendToken(String token) {
 
         Optional<UserEntity> userEntityOptional = userRepository.findById(Long.parseLong(userId));
 
+        if (!userEntityOptional.isPresent()) {
```

Comment:
User orElseThrow, like this:
UserEntity userEntity = userRepository.findById(Long.parseLong(userId)).orElseThrow(ExceptionSupplier.userNotFound);

And then no need for userEntityOptional.get()

Reply:
Done.

Reply:
I like it now :) 👍